# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1296366476 28800
#      Sat Jan 29 21:47:56 2011 -0800
# Node ID d5f50ea57869da10906151c6b8d0077f8d43867e
# Parent  f1ab2a12b4f45b651c52f090dfd90d5b53f40fbb
Add powerset function to documentation.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,12 @@
+2011-01-29  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/set.txi: Add powerset function to documentation.
+
 2011-01-29  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/system.txi: Add S_ISSOCK function to documentation.
 
 2011-01-29  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/system.txi: Add S_ISBLK and related functions to
 	documentation.
diff --git a/doc/interpreter/set.txi b/doc/interpreter/set.txi
--- a/doc/interpreter/set.txi
+++ b/doc/interpreter/set.txi
@@ -50,8 +50,10 @@ computes the union of the two sets.
 
 @DOCSTRING(union)
 
 @DOCSTRING(intersect)
 
 @DOCSTRING(setdiff)
 
 @DOCSTRING(setxor)
+
+@DOCSTRING(powerset)
diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-01-29  Rik  <octave@nomad.inbox5.com>
+
+	* set/powerset.m: Improve docstring.
+
 2010-01-29  Rik  <octave@nomad.inbox5.com>
 
 	* miscellaneous/copyfile.m, miscellaneous/movefile.m,
 	miscellaneous/tempname.m: Improve docstrings
 
 2010-01-29  Rik  <octave@nomad.inbox5.com>
 
 	* deprecated/module.mk, image/module.mk: Deprecate saveimage.m.
diff --git a/scripts/set/powerset.m b/scripts/set/powerset.m
--- a/scripts/set/powerset.m
+++ b/scripts/set/powerset.m
@@ -15,17 +15,17 @@
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {} powerset (@var{a})
 ## @deftypefnx {Function File} {} powerset (@var{a}, "rows")
 ##
-## Return the cell array of all subsets of the set @var{a}.
+## Return a cell array containing all subsets of the set @var{a}.
 ##
 ## @end deftypefn
 ## @seealso{unique, union, setxor, setdiff, ismember}
 
 function p = powerset (a, byrows_arg)
 
   byrows = false;
 
